Project.toml

```diff
@@ -4,6 +4,7 @@ authors = ["Thomas Breuer <sam@math.rwth-aachen.de>", "Sebastian Gutsche <gutsch
 version = "0.6.2"
 
 [deps]
+Downloads = "f43a241f-c20a-4ad4-852c-f6b1247861c6"
```

Comment:
Perhaps also add a compat entry?

Reply:
oh yes, I forgot that. Then just "1.5.1" (i.e. the current version)? Otherwise, I do not really know how to forcefully downgrade my environment for testing

Reply:
Fine by me

Reply:
Apparently, I have been using 1.4.2 the entire time. Thus this is now in the compat entry. (Why i cannot upgrade to the current 1.5.1 is another story)